### DISCUSSION TITLE
In Greek mythology, who is the goddess of wisdom and warfare? :smiling_imp:

### DISCUSSION THREAD

**Discussion by peymansf2000** [ParanoidUser/yolo#1271] [Galaxy Trivia] [answered]
Bonus: Which planet is known as the "Red Planet" and is often referred to as Earth's "sister planet"?

**Comment by tomerf** [comment, marked as answer]
Athena is the Greek Goddess of Wisdom and War.

Bonus: While Mars is called the red planet, Venus is known as Earth's sister planet.

**Reply by peymansf2000** [reply]
:tada: :tada:
